torch/_inductor/select_algorithm.py

```diff
@@ -744,15 +732,16 @@ def __init__(self, msg, choice: ChoiceCaller, inputs_str):
 class AlgorithmSelectorCache(PersistentCache):
     def __call__(self, name, choices: List[ChoiceCaller], input_nodes, layout):
         # TODO(nmacchioni): remove once CI tests are fixed
+        print(f"enter AlgorithmSelectorCache, {len(choices)=}")
         choices = [choice for choice in choices if choice is not None]
         if len(choices) == 0:
             raise RuntimeError(
                 "No choices to select, please consider adding ATEN into max_autotune_gemm_backends "
                 "config (defined in torch/_inductor/config.py) to allow at least one choice. "
             )
 
-        if len(choices) == 1:
-            return choices[0].output_node()
+        # if len(choices) == 1:
+        #     return choices[0].output_node()
```

Comment:
Need to make sure that workspace_size is fetched via the benchmark process.

Reply:
I see. I guess, this means that even one-option autotuning (like in the ATen-only cases) will run the kernel / invoke the subprocess etc.? I'm wondering if this can degrade compilation time. Likely the cache will be hit on subsequent attempts, though.